runtime/lua/vim/filetype.lua

```diff
@@ -125,6 +125,12 @@ local extension = {
     end
     return "cpp"
   end,
+  cc = function()
```

Comment:
Even though they’re not used in this function can you add the `path` and `bufnr` parameters to the function signature (and can you also do the same for the `cpp` function just above this one)?

Reply:
done.